.github/workflows/flags.yml

```diff
@@ -69,14 +69,15 @@ jobs:
           ghc: ${{ matrix.ghc }}
           os:  ${{ runner.os }}
 
-      - name: Build `hls-graph` with flags
-        run: cabal v2-build hls-graph --flags="embed-files stm-stats"
-
-      - name: Build `ghcide` with flags
-        run: cabal v2-build ghcide --flags="ghc-patched-unboxed-bytecode test-exe executable bench-exe ekg"
+      - name: Configue extra flags for some components
```

Comment:
ghc-patched-unboxed-bytecode and bench-exe flags seem to be some leftovers - they don't exist in ghcide.cabal anymore, so I just removed them from here.

The idea behind using cabal configure is that we do only single cabal build with pedantic flags enabled - and don't skip pedantic when building hls-graph and ghcide.